allennlp/data/token_indexers/pretrained_transformer_indexer.py

```diff
@@ -106,6 +106,8 @@ def tokens_to_indices(self, tokens: List[Token], vocabulary: Vocabulary) -> Inde
     def indices_to_tokens(
         self, indexed_tokens: IndexedTokenList, vocabulary: Vocabulary
     ) -> List[Token]:
+        self._add_encoding_to_vocabulary_if_needed(vocabulary)
```

Comment:
There was an issue that came up a little while ago that made me think we should also add this line to `count_vocab_items`.  What do you think?  I don't think it would hurt anything to have it there too, and it fixes a potential bug.

Reply:
`count_vocab_items()` doesn't receive a `Vocabulary`, so it can't be done there.

We still have a problem with this whole setup anyways, because there is no guarantee that we pass in the same vocab object every time, but we remember whether we've already done this in the indexer class. This only happens to work, but it's not right.

But this is not the PR to fix it.